QUICKSTART.md

```diff
@@ -0,0 +1,351 @@
+
+```bash
+kubectl ws tree
+```
+
+which should yield something like:
+
+```console
+.
+└── root
+    └── compute
+```
+
+## 2. Install and run **KCP-Edge**
+
+Download the kcp **KCP-Edge** binaries and scripts into a `kcp-edge` subfolder in your current working directory using the following command:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/kcp-dev/edge-mc/main/hack/install-kcp-edge.sh) --version v0.1.0 --folder $(pwd)/kcp-edge --create-folder
+export PATH="$PATH:$(pwd)/kcp-edge/bin"
+```
+
+Start **KCP-Edge** with the following command:
+
+```shell
+kcp-edge start --user kit
+```
+
+Check that `KCP-Edge` controllers are running with the following command:
+
+```shell
+ps aux | grep -e mailbox-controller -e placement-translator -e scheduler
+```
+
+which should yield something like:
+
+```console
+user     1892  0.0  0.3 747644 29628 pts/1    Sl   10:51   0:00 mailbox-controller --inventory-context=root --mbws-context=base -v=2
+user     1902  0.3  0.3 743652 27504 pts/1    Sl   10:51   0:02 scheduler -v 2 --root-user kcp-admin --root-cluster root --sysadm-context system:admin --sysadm-user shard-admin
+user     1912  0.3  0.5 760428 41660 pts/1    Sl   10:51   0:02 placement-translator --allclusters-context system:admin -v=2
+```
+
+Check that the Edge Service Provider Workspace (`espw`) is created with the following command:
+
+```shell
+kubectl ws tree
+```
+
+which should yield:
+
+```console
+.
+└── root
+    ├── compute
+    └── espw
+```
+
+## 3. Create a kcp-edge Inventory Management Workspace (IMW) and Workload Management Workspace (WMW)
+
+**IMW** are used by **KCP-Edge** to store *sync targets* and *placement* objects.
+Create an **IMW** named `imw-1` with the following command:
+
+```shell
+kubectl ws root
+kubectl ws create "imw-1"
+```
+
+**WMW** are used by **KCP-Edge** to store *workloads* amd *edge placement* objects.
+Create an **WMW** named `wmw-1` in a `my-org` workspace with the following command:
+
+```shell
+kubectl ws root
+kubectl ws create "my-org"
+kcp-edge --create_wmw wmw-1  # replaces: ensure-wmw.sh "wmw-1"
+```
+
+
+## 4. Example deployment of nginx workload into a kind local cluster
+
+ 
+### a. Stand up a local florin kind cluster
+
+To install kind in your environment follow the instructions [here](https://kind.sigs.k8s.io/). To use kind, you will also need to [install docker](https://docs.docker.com/get-docker/).
+
+```shell
+kind create cluster --name florin
+```  
+
+### b. Create a sync target, placement and edge syncer for onboarding the created florin edge cluster
+
+Create a syncTarget and location inventory objects to represent the edge cluster (`florin`):
+
+```shell
+kcp-edge --create_inv_item florin  env=prod    # replaces ensure-location.sh florin  env=prod
+```
+
+The following commands list the objects that were created:
+
+```console
+$ kubectl get locations,synctargets
+NAME                                RESOURCE      AVAILABLE   INSTANCES   LABELS   AGE
```

Comment:
The column for LABELS with empty content is confusing, but _not_ incorrect for TMC Location objects.  We should probably compose a non-confusing `kubectl get` command with custom columns.